packages/frontend-2/components/viewer/anchored-point/Thread.vue

```diff
@@ -147,10 +144,29 @@
               </div>
             </div>
             <ViewerAnchoredPointThreadNewReply
-              v-if="!modelValue.archived && canReply"
+              v-if="
+                !modelValue.archived &&
```

Comment:
Getting a bit too long/complicated for the template. Probably a good idea to move this to a computed.